src/main/java/seedu/storemando/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.storemando.logic.parser;
+
+import static seedu.storemando.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.storemando.logic.commands.SortCommand;
+import seedu.storemando.logic.commands.SortExpiryDateCommand;
+import seedu.storemando.logic.commands.SortQuantityCommand;
+import seedu.storemando.logic.parser.exceptions.ParseException;
+
+public class SortCommandParser implements Parser<SortCommand> {
+
+    public static final String QUANTITY_KEYWORD = "quantity";
+    public static final String EXPIRYDATE_KEYWORD = "expirydate";
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ListCommand
+     * and returns a ListCommand object for execution.
+     */
+    public SortCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim().toLowerCase();
+        if (trimmedArgs.equals(QUANTITY_KEYWORD)) {
```

Comment:
Personally I will prefer trimmedArgs = trim()
Then in the condition, we use equals ignore case instead.

Reply:
sure will do!